.ci/openshift-ci/test.sh

```diff
@@ -48,7 +65,7 @@ END
 }
 
 deployChe() {
-  chectl server:deploy  --che-operator-cr-patch-yaml=custom-resources.yaml --telemetry=off --workspace-engine=dev-workspace --platform=openshift --installer=operator --batch
+  chectl server:deploy  --che-operator-cr-patch-yaml=custom-resources.yaml --telemetry=off --platform=openshift --installer=operator --batch
```

Comment:
telemetry should be turned off by config like

```bash
  # turn off telemetry
  mkdir -p ${HOME}/.config/chectl
  echo "{\"segment.telemetry\":\"off\"}" > ${HOME}/.config/chectl/config.json
```

so we don't need to append --telemetry=off on every command

Reply:
The "chectl" used only in one place so in my eyes using additional "--telemetry=off" flag looks more clear than
```
  # turn off telemetry
  mkdir -p ${HOME}/.config/chectl
  echo "{\"segment.telemetry\":\"off\"}" > ${HOME}/.config/chectl/config.json
  ```